docs/build/support.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Support
+description: A comprehensive guide on how to reach out for support
+---
+
+import ColoredText from '/src/components/ColoredText';
+
+# Support
+
+This section explains how you can get support and stay up-to-date with Celo developer news.
+
+:::tip
+Help us improve by sharing your feedback on missing tools, unclear sections, or any challenges you encounter by filling out this [Feedback Form](https://forms.gle/14t1bhQyGDm7YQNG8).
+:::
+
+## How to Get Support?
+
+- For general questions, join our <ColoredText>[Discord](https://discord.com/invite/celo)</ColoredText> and claim the developer role to ask questions in our #general-dev channel. Our moderators can provide guidance and answers.
+
+- For advanced questions, create issues and participate in <ColoredText>[GitHub Discussions](https://github.com/orgs/celo-org/discussions)</ColoredText>.
+
+For the latest updates, sign up for the <ColoredText>[Celo Signal Mailing List](https://share.hsforms.com/1Qrhush1vSA2WIamd_yL4ow53n4j)</ColoredText> and follow CeloDevs on <ColoredText>[Twitter](https://x.com/CeloDevs)</ColoredText>.
```

Comment:
update link here to register as a builder, as signal is not providing technical updates. 

Created a ticket to create a signup workflow for DevDesk in Hubspot.